engineering-team.html

```diff
@@ -157,9 +168,37 @@ <h3>Elixir</h3>
 <!-- Script for static navbar and footer -->
 <script src="assets/js/navigation.js"></script>
 <script src="assets/js/functions.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+
```

Comment:
We don't need this line right? So shall we remove it? @gajindubandara 

```suggestion
```

Reply:
Sure!